mne/time_frequency/psd.py

```diff
@@ -95,7 +105,7 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
 
     Returns
     -------
-    psds : ndarray, shape (..., n_freqs) or
+    psds : ndarray, shape (..., n_freqs) or (..., n_windows, ..., n_freqs)
```

Comment:
Do you mean that `n_epochs, n_channels, n_windows, n_freqs` would be better in the full case? I prefer `n_epochs, n_windows, n_channels, n_freqs` - seems a bit more logical when windows are treated similarily to epochs (but lower in hierarchy).